docs/running/cloud/gce.rst

```diff
@@ -0,0 +1,57 @@
+.. _runningGCE:
+
+Running in Google Compute Engine (GCE)
+======================================
+
+Toil supports a provisioner with Google, and a :ref:`googleJobStore`. To get started, follow instructions
+for :ref:`prepareGoogle`.
+
+.. _googleJobStore:
+
+Google Job Store
+----------------
+
+To use the Google Job Store you will need to set the
+``GOOGLE_APPLICATION_CREDENTIALS`` environment variable by following `Google's instructions`_.
+
+Then to run the sort example with the Google job store you would type ::
+
+    $ python sort.py google:my-project-id:my-google-sort-jobstore
+
+Running a Workflow with Autoscaling
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The steps to run a GCE workflow are similar to those of AWS (:ref:`Autoscaling`), except you will
+need to explicitly specify the ``--provisioner gce`` option which otherwise defaults to ``aws``.
+
+#. Download :download:`sort.py <../../../src/toil/test/sort/sort.py>`.
+
+#. Launch the leader node in GCE using the :ref:`launchCluster` command. ::
+
+    (venv) $ toil launch-cluster gce-sort --provisioner gce --leaderNodeType n1-standard-1 \
+              --keyPairName <ssh-keyName> --boto <botoDir> --zone us-west1-a
+
+   The ``--boto`` option is necessary to talk to an AWS jobstore. This also requires that your aws credentials
+   are actually saved in your ``.boto`` file.
```

Comment:
A link to the boto page on the `.boto` file might be good here.

Reply:
since this is going to change soon anyway, I don't really want to / think it matters